rxcomponents/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+apply from: "$rootDir/java.gradle"
+apply plugin: 'kotlin'
+apply from: "$rootDir/kotlin.gradle"
+apply from: "$rootDir/spek.gradle"
+
+group = 'com.github.theREDspace.android-utils'
```

Comment:
I think this group tag is no longer needed as of another commit from me

Reply:
removing